src/components/builds/BuildsTable.tsx

```diff
@@ -90,6 +91,10 @@ const styles = theme =>
     infoIcon: {
       color: theme.palette.action.active,
     },
+    link: {
+      // default palette.primary.main colors
+      color: theme.palette.mode === 'dark' ? '#90caf9' : '#1976d2',
```

Comment:
Can it be just `theme.palette.primary.main`?

Reply:
We redefined this color in palette, so if i use it from theme, i get black color for link (both light and dark theme).